xapian-applications/omega/query.cc

```diff
@@ -1307,6 +1309,15 @@ eval(const string &fmt, const vector<string> &param)
 		value = str(collapsed);
 		break;
 	    }
+        case CMD_contains: {
+        size_t pos = args[1].find(args[0]);
+        if(pos!=string::npos){
+            value = str(pos);
+        } else {
+            value = "";
+        }
+        break;
+        }
 	    case CMD_csv:
```

Comment:
Generally looks good to me, but there are some minor issues:

The indentation is off here (and above).  I suspect you have tabs set to display with width 4 - you need to have them display as width 8 for Xapian code.

You also should put whitespace around operators, etc for readability.  Xapian coding conventions for the `if` would make it:

```
        if (pos != string::npos) {
```

And `value` will be empty by default, so there's no need for the `else` branch here.